### PR TITLE
Add feature introduction popup

### PR DESCRIPTION
**Pull request by 1orZero**
This branch depends on: `derek/20230521-notion-integration`, please merge it first.

- Integrate the Notion Features page.
- Add the description for each feature in Chinese Traditional and English
- Insert the banner [NEW new features page] on the New Conversation page.
- Support in-site notion page-link redirect

Need to add env var 
`NOTION_FEATURES_DATABASE_ID`

### CODE REVIEW

File: .env.local.example [hunk 1]
```diff
@@ -12,5 +12,12 @@ STRIPE_WEBHOOK_SECRET=KEY
 AZURE_SPEECH_KEY=KEY
 AZURE_SPEECH_REGION=REGION
 
+# Notion
+NOTION_SECRET_KEY=KEY
+NOTION_NEWS_DATABASE_ID=KEY
+NOTION_FEATURES_DATABASE_ID=KEY
+
+# Image generation
+STABILITY_AI_KEY=KEY
```
Comment (thejackwu) [inline .env.local.example]: Since we've switched to MJ, we can remove this env

File: components/Features/FeaturePage.tsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+      .filter((item: string) => item !== '');
+  };
+
+  useEffect(() => {
+    fetchPageData(pageId);
+  }, [pageId]);
+  if (recordMap) {
+    {
+      /* <div className="text-center font-bold">{title}</div> */
+    }
```
Comment (thejackwu) [inline components/Features/FeaturePage.tsx]: I think we can remove this comment?

File: pages/api/notion/[id].ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+
+import { NotionAPI } from 'notion-client';
+
```
Comment (thejackwu) [inline pages/api/notion/[id].ts]: Let's add in the edge function config to improve performance.

File: pages/api/notion/[id].ts
Comment (thejackwu) [inline pages/api/notion/[id].ts:1]: Let's also add Cache control to improve loading performance. 

https://vercel.com/docs/concepts/functions/edge-functions/edge-caching

File: pages/api/notion/features/index.ts
Comment (thejackwu) [inline pages/api/notion/features/index.ts:1]: Let's also add caching to this to improve performance, and reduce number of API calls. I think a 60 mins cache expiration is safe

File: pages/api/notion/news/index.ts
Comment (thejackwu) [inline pages/api/notion/news/index.ts:1]: Same as above

File: public/locales/zh-Hant/chat.json [hunk 4]
```diff
@@ -48,5 +48,6 @@
   "If you like this project, please support us by subscripting to our Pro plan!": "如果你喜歡這個APP,請通過Pro月費計劃來支持這個項目的可持續發展,每月只需美金$9.99！",
   "Please share your thoughts by completing a brief survey.": "請參加問卷調查，讓我們更懂您的需求",
   "Remaining Credits": "剩餘點數",
-  "Click to download image": "點擊下載"
-}
\ No newline at end of file
+  "Click to download image": "點擊下載",
+  "Check out out new features page!": "歡迎查看我們的新功能頁面！"
```
Comment (thejackwu) [inline public/locales/zh-Hant/chat.json]: I think we can make it more attractive. Something like `歡迎瀏覽我們的功能介紹，這些功能會讓您更快上手!`

**Review by thejackwu — CHANGES_REQUESTED** [review]

**Comment by 1orZero** [comment]
Updated the **api/notion/[id]** route to use serverless function and added cache for 60mins for it. It should be really fast now. @thejackwu

**Review by thejackwu — APPROVED** [review]

**Pull request merged**